lib/rss/rss.rb

```diff
@@ -25,7 +25,7 @@ def w3cdtf(date)
             \s*\z/ix =~ date and (($5 and $8) or (!$5 and !$8))
           datetime = [$1.to_i, $2.to_i, $3.to_i, $4.to_i, $5.to_i, $6.to_i]
           usec = 0
-          usec = $7.to_f * 1000000 if $7
+          usec = $7.to_f * 1_000_000 if $7
```

Comment:
```suggestion
          usec = $7.to_f * 1000000 if $7
```

Not in `Style/StringLiterals`.
Not in `Style/StringLiterals`.